src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,101 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import java.util.Objects;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private static final double INCREASE_SIZE = 1.5;
+    private int size;
+    private Object[] array;
+
+    public ArrayList() {
+        this.size = 0;
+        this.array = new Object[DEFAULT_SIZE];
+    }
+
     @Override
     public void add(T value) {
-
+        if (array.length == size) {
+            resize();
+        }
+        array[this.size] = value;
+        size++;
     }
 
     @Override
     public void add(T value, int index) {
-
+        if (index < 0 || index >= size) {
```

Comment:
here (ONLY ON THIS LINE) '=' not allowed, it will be  'if (index < 0 || index >= size)'